core/core/validators.py

```diff
@@ -2,48 +2,41 @@
 from datetime import datetime
 
 from core.database import Database
-from core.misc import (
-    InvalidBirthdayError,
-    InvalidEmailError,
-    InvalidNameError,
-    InvalidPhoneError,
-    InvalidPhoneLengthError,
-)
 
+class Validator:
+    def validate_existing_contact(name):
+        database = Database()
+        record = database[name]
 
-def validate_existing_contact(name):
-    database = Database()
-    record = database[name]
+        if record:
+            return record
+        else:
+            raise KeyError
```

Comment:
We should return true or false. You already raise exception in add_contact service